src/data.js

```diff
@@ -411,6 +411,9 @@ export class RepoAPI {
 
         return new Promise((resolve, reject) => {
             let name = repository.name
+            if (name == null) {
```

Comment:
Please use always `===` and `!==` instead of `==` and `!=`.

But in this case the best method to set a default is probably just:

```javascript
let name = repository.name || ""
```